.env.example

```
@@ -9,6 +9,11 @@
 # -----------------------------------------
 # Get your API key from trymaple.ai after upgrading to Pro/Team/Max
 MAPLE_API_KEY=your-maple-api-key-here
+# Sanctum is Maple-only for LLM inference
+LLM_PROVIDER=maple
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LLM_PROVIDER key should go before the MAPLE_API_KEY key

(UnorderedKey)